modules/feedreader/widget.go

```diff
@@ -51,13 +51,14 @@ func rotateShowType(showtype ShowType) ShowType {
 }
 
 func getShowText(feedItem *FeedItem, showType ShowType) string {
-	returnValue := feedItem.item.Title
+	title := strings.Replace(feedItem.item.Title, "\n", " ", -1)
```

Comment:
Thanks for your PR to fix my issue! Apologies, haven't had a chance to test this yet, but could this lead to multiple spaces if the original title has a space either before or after the line break?
E.g. 
( where _ is a space character)
```
This is a title with_
a break
```

Might become:

```
This is a title with  a break
```

Reply:
Hey Adam 👋,

Yes, you are totally right, if the title has spaces and `\n` characters it will create an even spacier string.

Then, I came up with another solution using regex expressions. 

I committed those changes! 👀

Reply:
Awesome! 👍 

![image](https://user-images.githubusercontent.com/71472057/124848277-a6e36e80-dff0-11eb-8f10-4ad5df5e0622.png)
💚